BUILD.bazel

```diff
@@ -37,8 +38,8 @@ copy_file(
 py_binary(
     name = "protoc-gen-nanopb",
     srcs = [
-        ":protoc-gen-nanopb.py",
         "generator/nanopb_generator.py",
+        ":protoc-gen-nanopb.py",
```

Comment:
Was the reordering of srcs here your formatting change or buildifier? I might have forgotten to format the plugin in my previous PR, though I'm unsure. In any case the plugins *should* be autoformatted to prevent thrash.

Reply:
I think gazelle caught it for me. I didn't manually move it!

Reply:
Yeah. I just tried manually moving it above and then re-ran gazelle and it moved it down.

Reply:
Fair enough. Well, that's my fault then. Best to leave it as is and we can put a CI check in there at some point to prevent new changes that aren't formatted properly.

Reply:
Just to be clear; you want me to revert this line change?

Reply:
If it's formatted with buildifier, then don't revert it.